code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -574,14 +577,20 @@
 		icon_state = "[lasercolor]ed209[on]"
 	var/threat = C.assess_threat(src)
 	C.SetStuttering(10 SECONDS)
-	C.Weaken(10 SECONDS)
-	add_attack_logs(src, C, "stunned")
+	C.adjustStaminaLoss(60)
+	baton_delayed = TRUE
+	addtimer(CALLBACK(C, .proc/KnockDown, 10 SECONDS), 2.5 SECONDS)
+	addtimer(CALLBACK(src, .proc/reset_baton_cooldown), BATON_COOLDOWN)
```

Comment:
you can make use of `VARSET_CALLBACK` here, so that you don't need a basically empty proc.